### ISSUE TITLE
Restructure vignette: handling, plotting, analysis

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/epicontacts#33]
As it currently stands, the vignette does a good job of showing functionalities but is a bit hard to follow. It reads like a catalog, which is more the purpose of the manual. We could probably make information more accessible by re-structuring the content. I would think:
1. Introduction & overview (i.e. what is this package for?)
2. Installation
3. Getting your data into _epicontact_
4. Handling data
5. Plotting data
6. Analysis

@vpnagraj @nistara @tc13 @finlaycampbell What do y'all think?

**Comment by vpnagraj** [comment]
implemented new vignette structure ... see ac963f9
